web/src/features/chat/components/ChatMessage.tsx

```
@@ -115,9 +115,10 @@ const renderRichContent = (text: string, isThoughts = false) => {
             )
         } else {
             flushList(index)
-            const processedLine = line
-                .replace(/^\*\*Overview:\*\*\s*/i, '')
-                .replace(/^Overview:\s*/i, '')
+            const processedLine = line.replace(
+                /^(\*\*Overview:\*\*|\*\*Overview\*\*|Overview:?)\s*[\-–—]?\s*/i,
```

ESLint

[error] Unnecessary escape character: \-.

(no-useless-escape)